tests/credential_plugins_test.py

```diff
@@ -137,13 +137,19 @@ class TestDelineaImports:
     library, so these tests are designed to fail if these wind up using the
     fallback import."""
 
-    def test_dsv_import(self):
-        from awx_plugins.credentials.dsv import SecretsVault  # noqa
+    def test_dsv_import(self) -> None:
+        from awx_plugins.credentials.dsv import SecretsVault  # noqa: F401
+
         # assert this module as opposed to older thycotic.secrets.vault
         assert SecretsVault.__module__ == 'delinea.secrets.vault'
 
     def test_tss_import(self):
-        from awx_plugins.credentials.tss import DomainPasswordGrantAuthorizer, PasswordGrantAuthorizer, SecretServer, ServerSecret  # noqa
+        from awx_plugins.credentials.tss import (  # noqa: F401
+            DomainPasswordGrantAuthorizer,
+            PasswordGrantAuthorizer,
+            SecretServer,
+            ServerSecret,
+        )
```

Comment:
@chrismeyersfsu do these tests actually make sense in this form with in-function imports? Other smoke tests already import them, so that would influence the in-process module cache anyway.